backend/src/script.ts

```diff
@@ -32,4 +34,37 @@ const formatReview = (data: any): Review => ({
   photos: data.photos,
 });
 
+const formatLandlord = ({
+  id,
+  name,
+  contact,
+  avgRating,
+  photos,
+  reviews,
+  properties,
+}: // eslint-disable-next-line @typescript-eslint/no-explicit-any
+any): LandlordWithId => ({
+  id: id.toString(),
+  name,
+  contact,
+  avgRating: avgRating.length > 0 ? Number.parseFloat(avgRating) : 0,
```

Comment:
This can be `avgRating: Number.parseFloat(avgRating || 0)`

Same for the rest, for example `photos: photos || []`